src/http/api/views/account.follows.viewer.ts

```diff
@@ -0,0 +1,228 @@
+import type { Account } from 'account/account.entity';
+import { getAccountHandle } from 'account/utils';
+import type { Knex } from 'knex';
+import type { FedifyContextFactory } from '../../../activitypub/fedify-context.factory';
+
+/**
+ * Maximum number of follow accounts to return
+ */
+const FOLLOWS_LIMIT = 20;
+
+interface AccountInfo {
+    id: string;
+    name: string;
+    handle: string;
+    avatarUrl: string;
+    isFollowing: boolean;
+}
+
+interface AccountFollowsView {
+    accounts: AccountInfo[];
+    total: number;
+    next: string | null;
+}
+
+interface AccountRow {
+    id: number;
+    ap_id: string;
+    name: string;
+    username: string;
+    avatar_url: string;
+}
+
+export class AccountFollowsViewer {
+    /**
```

Comment:
This JSDoc doesn't have all the arguments - but I think we can just remove it, it doesn't really provide any extra info